### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by Kwentsir**
In this milestones, the following were implemented:

- Used primary key & foreign key mechanism for joining tables.

- Queried multiple tables.

- Prepared complex queries that answer analytical questions.
mtquery1

mtquery2

mtquery3

mtquery4

mtquery5

mtquery6

mtquery7

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -9,4 +9,23 @@ neutered BOOL,
...
 species varchar(255),
 PRIMARY KEY(id)
-);
\ No newline at end of file
+species_id int,
+owner_id int
+);
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
```
Comment (arslanbisharat) [inline schema.sql]: - [ ] Please make sure `id` is an integer and never null.

File: schema.sql [hunk 2]
```diff
@@ -9,4 +9,23 @@ neutered BOOL,
...
+    full_name varchar(100),
+    age int
+);
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
```
Comment (arslanbisharat) [inline schema.sql]: - [ ]  Please make sure `id `is an integer and never null.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Kwentsir, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi @Kwentsir  👍🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**